index.php

```diff
@@ -6,17 +6,14 @@ class FishType implements JsonSerializable {
 
     private $character;
 
-    private $phMin;
-
-    private $phMax;
+    private $phRange;
 
     private $cost;
 
-    public function __construct(string $name, string $character, float $phMin, float $phMax, float $cost) {
+    public function __construct(string $name, string $character, array $phRange, float $cost) {
```

Comment:
I was thinking about introducing an object that represents a Range, and can tell if range A and B intersect. 
Array isn't a good idea here because you implementation requires a specific step for the range. So it can't be any array.